source/catalog.html

```diff
@@ -218,7 +219,7 @@ <h2 class="additional__title">Дополнительные товары</h2>
           <a class="button additional__button" href="#">Заказать</a>
         </li>
       </ul>
-      <p class="present">Закажите все и получите чехол для кота в подарок!</p>
+      <p class="additional__text present">Закажите все и получите чехол для кота в подарок!</p>
```

Comment:
Почему не так? Логично же )
```suggestion
      <p class="additional__present present">Закажите все и получите чехол для кота в подарок!</p>
```

Reply:
Согласна)